test/rekt/features/broker/control_plane.go

```diff
@@ -580,6 +581,19 @@ func getBroker(ctx context.Context, t feature.T) *eventingv1.Broker {
 	return broker
 }
 
+func copyBroker(ctx context.Context, srcBroker *eventingv1.Broker, toName string) (*eventingv1.Broker, error) {
+	broker := &eventingv1.Broker{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:        toName,
+			Labels:      srcBroker.Labels,
+			Annotations: srcBroker.Annotations,
+		},
+		Spec: srcBroker.Spec,
+	}
```

Comment:
nit: a more concise approach?
```suggestion
	broker := srcBroker.DeepCopy()
	broker.Name = toName
```

Reply:
or maybe it doesn't work because we would have the uuid, etc set on the metadata which wouldn't be correct ?

Reply:
However, I would DeepCopy() the spec `*srcBroker.Spec.DeepCopy()` while setting it to Spec?

Reply:
> or maybe it doesn't work because we would have the uuid, etc set on the metadata which wouldn't be correct ?
yeah this was my concert too :/
But will use `*srcBroker.Spec.DeepCopy()` :+1: